lib/plugins/exchange/ccxt.js

```diff
@@ -49,4 +54,36 @@ function calculatePrice (side, amount, orderBook) {
   throw new Error('Insufficient market depth')
 }
 
-module.exports = { trade }
+function _getMarkets (exchangeName, availableCryptos) {
+  try {
+    const exchange = new ccxt[exchangeName]()
+    const cryptosToQuoteAgainst = ['USDT']
+    const currencyCodes = _.concat(_.map(it => it.code, currencies), cryptosToQuoteAgainst)
+
+    return exchange.fetchMarkets()
+      .then(_.filter(it => (it.type === 'spot' || it.spot)))

```

Comment:
We are using market trading for some exchanges. I think at this point we'll need to filter depending on exchange.

Reply:
I investigated further and correct me if I'm wrong on this. I believe there's a misunderstanding between order types and market types. We execute trading orders with both market and limit orders, but these orders are executed on the spot market of the respective exchanges. Other options for market are `future` and `swap` (for our supported exchanges), but I don't think we do trades on those markets